ts/utils/payment.ts

```diff
@@ -8,3 +20,24 @@ export function formatPaymentAmount(amount: number): string {
   // We need to divide by 100 to get euro from euro-cents
   return I18n.toNumber(amount / 100, { precision: 2 });
 }
+
+/**
+ * Converts a NoticeNumber coming from a Message to an RptId needed by PagoPA
+ */
+export function getRptIdFromNoticeNumber(
```

Comment:
@fpersico @fgiobergia @gunzip @LsKoder  is this the right way to convert the data coming from a message to what is expected by PagoPA?

Reply:
`RptId` [has been defined](https://github.com/teamdigitale/italia-ts-commons/blob/e395e75cf40fe37b362c4858f0279d75f26c20b1/src/pagopa.ts#L360) to contain the payment notice number and the organization fiscal code (the two fields that identify a payment). 

What you are using here is the payment notice number alone (`noticeNumber`), but you also need to get the organization fiscal code (of type [`OrganizationFiscalCode`](https://github.com/teamdigitale/italia-ts-commons/blob/e395e75cf40fe37b362c4858f0279d75f26c20b1/src/strings.ts#L143) in order to build a valid `RptId`. 

`RptIdFromString` is a utility function that converts a string in the format `${fiscalCode}${noticeNumber}` into an RptId (which is then passed to the proxy)